### ISSUE TITLE
Remove type prefix of the suggested subject message

### ISSUE THREAD

**Issue by mayconfsousa** [leoforfree/cz-customizable#137]
The function **getPreparedCommit** should return the suggested message without the type prefixes (fix:, feat:,...).

```js
// questions.js
default: getPreparedCommit('subject'),
```

Current behavior:

1. Previous commit subject: "fix: my message"
2. Current commit type: "fix"
3. Current commit subject (suggestion): "fix: my message"
4. Current commit subject after accept the suggestion: "fix: my message"
5. Final commit subject "fix: fix: my message"

**Comment by storageddd** [comment]
Hello! I created a pull-request which will solve this problem.
https://github.com/leoforfree/cz-customizable/pull/132

**Comment by Alexa905** [comment]
+1

**Comment by flyerH** [comment]
@leonardoanalista Hello, can you look this problem? If you don't have time, maybe I can help.

**Comment by yiqianglin** [comment]
> @leonardoanalista Hello, can you look this problem? If you don't have time, maybe I can help.

@flyerH +1，is there any solution？

**Comment by flyerH** [comment]
> > @leonardoanalista Hello, can you look this problem? If you don't have time, maybe I can help.
> 
> @flyerH +1，is there any solution？

fork & fix

**Comment by leonardoanalista** [comment]
yes, I noticed this problem. I merge https://github.com/leoforfree/cz-customizable/pull/132 if rebased
